src/Sulu/Component/SmartContent/Tests/Unit/Orm/DataProviderRepositoryTraitTest.php

```diff
@@ -55,11 +55,11 @@ public function testFindByFiltersIds()
         );
         $findByFiltersIdsReflection->setAccessible(true);
 
-        $query = $this->getMockBuilder(Query::class)
-            ->disableOriginalConstructor()
-            ->getMock();
-        $query->method('setMaxResults')->willReturn($query);
-        $query->method('getScalarResult')->willReturn([]);
+        $query = $this->prophesize(Query::class);
+
+        $query->setFirstResult(0)->willReturn($query);
+        $query->setMaxResults(Argument::any())->willReturn($query);
+        $query->getScalarResult()->willReturn([]);
```

Comment:
I have made that change already in #4932

Reply:
merged